src/client/api/events/sub_milestone_payload.c

```diff
@@ -3,11 +3,11 @@
 
 #include <stdlib.h>
 
-#include "client/api/events/sub_milestone_latest.h"
-#include "client/api/json_utils.h"
+#include "client/api/events/sub_milestone_payload.h"
+#include "client/api/json_parser/json_utils.h"
 #include "client/network/mqtt/mqtt.h"
 
-int parse_milestone_latest(char *data, milestone_latest_t *res) {
+int parse_milestone_payload(char *data, milestone_payload_t *res) {
```

Comment:
`res` needs validation, it could be NULL. `data` is validated by cJSON_Parse so it should be okay.